server/Transaction.java

```diff
@@ -6,186 +6,168 @@
     String confirmation = "";
     double amount = 0.0;
     boolean valid = false;
-    Account fromAccount;
-    Account toAccount;
-    Customer toCustomer;
+    newbank.server.Account fromAccount;
+    newbank.server.Account toAccount;
+    newbank.server.Customer toCustomer;
 
-    //move funds between personal accounts
-    public String moveFunds(CustomerID customer, NewBankClientHandler newBankClientHandler) {
+    // Moving funds between personal accounts.
+    public String moveFunds(newbank.server.Customer customer, newbank.server.NewBankClientHandler newBankClientHandler) {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it